tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -130,6 +130,9 @@ TILEDB_EXPORT int32_t tiledb_array_schema_evolution_drop_attribute(
 
 /**
  * Sets timestamp range in an array schema evolution
+ * The consecutive evolved schemas could have the same timestamps.
```

Comment:
What does `consecutive evolved schemas` mean here? Is this function setting the timestamp range for the output schema?

Reply:
In some unit tests, we evolve schemas one by one. Those schemas has a small chance to have the same timestamp on a faster machine(such as Apple M1). @lums658 solved this problem by adding 1 milliseond to the timestamp,